Nixie/Nixie.c

```diff
@@ -123,6 +123,27 @@ ISR(TIMER2_COMPA_vect)
 	PORTB &= ~TIME_MEAS; // Time Measurement
 } // ISR()
 
+
+/*-----------------------------------------------------------------------------
+  Purpose  : This routine fills the led_<x>[NR_LEDS] array.
+  Variables: ws2812b_red, ws2812b_green, ws2812b_blue : the byte to fill
+  Returns  : -
+  ---------------------------------------------------------------------------*/
+void ws2812b_fill_rgb(uint8_t ws2812b_red, uint8_t ws2812b_green, uint8_t ws2812b_blue)
+{
+	uint8_t i;
+	
+	for (i = 0; i < NR_LEDS; i++)
+	{
+		led_g[i] = ws2812b_green;
+		led_r[i] = ws2812b_red;
+		led_b[i] = ws2812b_blue;

```

Comment:
Oke, dus alle leds dezelfde kleur? Prima voor nu, straks nog maar eens kijken of we dit per led kunnen gaan doen.

Reply:
Yep, voor nu wel. Even maar de standaard van de IN-14 aangehouden ivm de IR codes.
Moet nog een routine komen om de leds individueel aan te sturen als ook de licht opbrengst de sturen.